CHANGELOG.md

```diff
@@ -26,3 +26,4 @@
 -   [86cujnfuc] API integration of Trip Listing Page.
 -   [86cuewzt9] API integration of Trip Details Section.
 
+-   [86cupmdug] Added error page, illustration and navigation buttons
```

Comment:
[86cur11g8] is the changelog to be added right?

Reply:
Resolved in ed36de56ec2a66f79e07fbba35626656834f4b98